sale_comment_template/README.rst

```diff
@@ -12,17 +17,41 @@ Two positions are available for the comments:
 
 Texts are copied on the invoice when you will create invoice from sale order.
 
-Dependencies
-------------
+Usage
+=====
 
-The modules `invoice_comment_template` and `base_comment_template` are
```

Comment:
There should be a section `Installation` similar to this one, indicating that this module depends on `invoice_comment_template` and `base_comment_template` in OCA/account-invoice-reporting.

Reply:
Added the <code>Installation</code> section